lib/smack/SmackRep.cpp

```diff
@@ -758,36 +935,99 @@ const Expr* SmackRep::declareIsExternal(const Expr* e) {
 }
 
 string SmackRep::getPtrType() {
```

Comment:
Consistency would be best: either use the method `getPtrType()`, or a constant with value `"ref"`, or the literal `"ref"`. But let's not mix and match method calls with literals as is currently.